rapidhash.h

```diff
@@ -73,7 +73,13 @@
  # endif
  # define RAPIDHASH_INLINE_CONSTEXPR RAPIDHASH_INLINE
  #endif
- 
+
+ #ifdef __has_attribute(no_inline)
+ # define RAPIDHASH_NO_INLINE __attribute__((noinline))

```

Comment:
I'm less familiar with portable C and C++, so let me know if this could/should be improved to support more compilers. I think visual studio uses `__declspec(noinline)` for example?